src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -19,6 +19,13 @@ attributes will be exported when
 variable will be set to `true`.
 ([#4892](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4892))
 
+* Added ability to export attributes corresponding to `LogRecord.EventId` as
+`event.id` and `event.name` and `LogRecord.CategoryName` as
+`dotnet.ilogger.category`. These attributes will be exported when
+`OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_CATEGORY_AND_EVENT_LOG_ATTRIBUTES` will be
+set to `true`.
+([#4925](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4925))
```

Comment:
I think its good to mention the full context here for people upgrading from pre-stable, where we used to export this as a diff. attribute.
Something like. versions before A used to export this as attribute ...
version B removed this.
version C (current), adds this under feature flag,. Note that the attribute names are changed as well.

(So end users will have easier time following the changelog, instead of having to open multiple PR links and understand what changed if upgrdaing from 1.6.0.beta to 1.7.0.beta..